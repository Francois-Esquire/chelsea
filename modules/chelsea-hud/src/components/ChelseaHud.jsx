import React from 'react';

export default function ChelseaHUD({ children }) {
  return (
    <React.Fragment>
      {children}
    </React.Fragment>
  );
}
