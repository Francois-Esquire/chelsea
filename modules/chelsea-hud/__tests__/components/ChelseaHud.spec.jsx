import React from 'react';
import { render } from '@testing-library/react';
import ChelseaHud from '../../src/components/ChelseaHud';

describe('ChelseaHud should render as expected', () => {
  it('module should render correct JSX', () => {
    const { asFragment } = render(<ChelseaHud />);
    expect(asFragment()).toMatchSnapshot();
  });
});
