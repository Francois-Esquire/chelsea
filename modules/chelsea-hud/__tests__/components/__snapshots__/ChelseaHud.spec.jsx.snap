// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChelseaHud should render as expected module should render correct JSX 1`] = `<DocumentFragment />`;
